alfresco-remote-testrunner/build.gradle

```diff
@@ -25,20 +28,35 @@ dependencies {
 
     shadowed 'commons-lang:commons-lang:2.6'
     shadowed "junit:junit:4.12"
-    shadowed 'org.apache.httpcomponents:fluent-hc:4.3.3'
+    relocated 'org.apache.httpcomponents:fluent-hc:4.3.3'
 
     testImplementation 'com.github.stefanbirkner:system-rules:1.16.1'
     testImplementation 'org.mockito:mockito-core:2.23.4'
 }
 
+task shadowRelocate(type: ShadowJar) {
+    classifier = "relocation"
+    from(sourceSets.main.output)
+    configurations = [project.configurations.relocated]
+    relocate 'org.apache.http.client.fluent', 'eu.xenit.testing.integrationtesting.internal.shadow.org.apache.http.client.fluent'
```

Comment:
When using a `ConfigureShadowRelocation` task, it is no longer necessary to explicitly configure relocation for some classes. It will already be handled by that task